tendrl/alerting/objects/definition/alerting.py

```diff
@@ -2,12 +2,46 @@
 data = """---
 namespace.tendrl.alerting:
   objects:
+    Definition:
+      enabled: True
+      help: "Definition"
+      value: _tendrl/definitions/alerting
+      list: _tendrl/definitions/alerting
+      attrs:
+        master:
+          help: Alerting definitions
+          type: String
     Config:
-      value: '/_tendrl/config/alerting'
-      data:
-        help: "The configurations path"
-        type: json
+      attrs:
+        data:
+          help: Configuration data of alerting for this Tendrl deployment
+          type: str
+        etcd_connection:
+          help: Host/IP of the etcd central store for this Tendrl deployment
+          type: str
+        etcd_port:
+          help: Port of the etcd central store for this Tendrl deployment
+          type: str
+        file_path:
+          default: /etc/tendrl/alerting/alerting.conf.yaml
```

Comment:
Will remove it. earlier all this fields were maintained in def file including the conf file path and description of conf fields with their defaults. But looks like they are no more required to be maintained. Will remove it